docs/src/content/docs/pt/getting-started.mdx

```diff
@@ -0,0 +1,174 @@
+
+O Starlight é um tema de documentação cheio de funcionalidades construído em cima do framework [Astro](https://astro.build).
+Este guia irá ajudá-lo a começar um novo projeto.
+Consulte as [instruções de instalação manual](/pt/manual-setup) para adicionar o Starlight a um projeto Astro já existente.
+
+## Início Rápido
+
+### Crie um novo projeto
+
+Crie um novo projeto Astro + Starlight executando o seguinte comando no seu terminal:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+npm create astro@latest -- --template starlight
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+pnpm create astro --template starlight
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn create astro --template starlight
+```
+
+</TabItem>
+</Tabs>
+
+Isto irá criar um novo [diretório de projeto](/pt/guides/project-structure/) com todos os arquivos e configurações necessárias para o seu novo site.
+
+:::tip[Veja-o em ação]
+Experimente o Starlight directamente no seu navegador:
+[abrir o template no StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
+:::
+
+### Inicie o servidor de desenvolvimento
+
+Quando estiver a trabalhar localmente, o [servidor de desenvolvimento do Astro](https://docs.astro.build/pt/reference/cli-reference/#astro-dev) permitirá visualizar seu trabalho e atualizar automaticamente o seu navegador após fazer alterações.
+
+Dentro do diretório do seu projeto, execute o seguinte comando para iniciar o servidor de desenvolvimento:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+npm run dev
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+pnpm dev
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dev
+```
+
+</TabItem>
+</Tabs>
+
+Será exibida uma mensagem no seu terminal com o URL para a visualização local.
+Abra esse URL para começar a navegar no seu site.
+
+### Crie conteúdo
+
+O Starlight está pronto para você adicionar novos conteúdos, ou trazer os seus arquivos já existentes!
+
+#### Formatos de arquivo
+
+O Starlight suporta a escrita de conteúdo em Markdown e MDX sem precisar de nenhuma configuração extra.
+Pode também adicionar suporte para Markdoc instalando a [integração experimental Astro Markdoc](https://docs.astro.build/pt/guides/integrations-guide/markdoc/).
+
+#### Adicione páginas
+
+Adicione novas páginas ao seu site criando arquivos `.md` ou `.mdx` em `src/content/docs/`.
+Utilize subdiretórios para organizar seus arquivos, e para criar múltiplos segmentos de caminho.
+
+Por exemplo, a seguinte estrutura de arquivos vai gerar páginas `example.com/ola-mundo` e `example.com/guides/faq`:
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - faq.md
+      - ola-mundo.md
+
+</FileTree>
+
+#### Frontmatter com segurança de tipos
+
+Todas as páginas do Starlight partilham um [conjunto comum de propriedades frontmatter](/pt/reference/frontmatter/) para controlar como a página aparece:
```

Comment:
Isn't the correct form "partilham _de_ um conjunto..."?

Reply:
Actually I believe they can both be used.
https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/regencias-partilhar-de-compartilhar-de/36757
Although "partilhar de" is more used for "dividing" something.